source/server-discovery-and-monitoring/server-discovery-and-monitoring.rst

```diff
@@ -838,6 +841,19 @@ are parsed from the ismaster response in the obvious way.
 
 .. _updates its view of the topology:
 
+Server Description Equality
+```````````````````````````
+
+For the purposes of updating topology description, two server descriptions
+MUST be considered equal if the data they contain which is parsed from
+the respective ismaster response is identical. If the parsed ismaster data
+is different, the server descriptions MUST be considered not equal.
```

Comment:
Let's be clearer that this excludes RTT and lastUpdateTime.

Reply:
Agreed. I was looking for something more explicit than "they have the same ismaster", which fields? Can we ignore some ismaster fields? This equality check will be happening every 10s in some cases, can we reduce the number of fields we need to check? Can we take a subset of fields and hash them for comparison?

Reply:
Hold up =) Because the equality check happens once every 10 seconds that's an argument for keeping it simple, not for optimizing it! Let's leave it up to maintainers how they want to check that fields are equal.

Reply:
Ok, well then let's at least enumerate which fields should not be considered.